features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/mobileapp/renderers/controllers/assets_router.js

```diff
@@ -36,7 +36,17 @@ var render = function (theme, data, meta, require) {
 
 	}
 
-	var lifecycleColors = {"Demote": "btn-blue", "Submit for Review": "btn-blue", "Publish": "btn-blue", "Unpublish": "btn-orange", "Deprecate": "btn-danger", "Retire": "btn-danger", "Approve": "btn-blue", "Reject": "btn-danger"};
+	var lifecycleColors = {
```

Comment:
This is merely a code formatting. Also, 'lifecycle' prefix is used in other variable names (e.g. `lifecycleState`, `lifecycleAvailableActionsButtons`)
PS: code no longer exists to correct.